### ISSUE TITLE
Doesnt train

### ISSUE THREAD

**Issue by Arkoprabho** [Arkoprabho/RNN-Tensorflow#1]
The network shows a consistent loss throughout epochs.

**Comment by Arkoprabho** [comment]
Check commit hash f807b9b9d52107cefa3e47004e3440eb9ee461b7
